tests/e2e/performance/csi_tests/test_pvc_bulk_clone_performance.py

```diff
@@ -71,50 +93,49 @@ def init_full_results(self, full_results):
         full_results.add_key("index", full_results.new_index)
         return full_results
 
-    @pytest.fixture()
-    def namespace(self, project_factory, interface_iterate):
-        """
-        Create a new project
+    def attach_multiple_pvc_to_pod_dict(self, pvc_list):
         """
-        proj_obj = project_factory()
-        self.namespace = proj_obj.namespace
-        self.interface = interface_iterate
+        Function to construct pod.yaml with multiple PVC's
 
-        if self.interface == constants.CEPHFILESYSTEM:
-            sc = "CephFS"
-        if self.interface == constants.CEPHBLOCKPOOL:
-            sc = "RBD"
+        Args:
+            pvc_list (list): list of PVCs to be attached to single pod
 
-        self.full_log_path = get_full_test_logs_path(cname=self)
-        self.full_log_path += f"-{sc}"
+        Returns:
+            pod_data (str): pod data with multiple PVC mount paths added
+
+        """
+        pods_list = []
+        for pvc_name in pvc_list:
```

Comment:
The function states that "Function to construct pod.yaml with multiple PVC's" as I understand you have multiple PVCs to one POD - correct me if I'm wrong. But here for every PVC you load a POD template and create a new pod. Either the function description is misleading or the creation of multiple pods is wrong.

Reply:
the function **can** connect multiple pvcs to one pod, in this case, i used it to connect only one pvc to each pod.

Reply:
If the function is getting only pvc_list can you show where the **can** can happen. From looking into the function it can't connect multiple PVC to one POD so please correct the description because the function can do only multiple pvc to multiple pods, there is no option to tell the function to attach all the PVC to one POD.